pithos-save.py

```diff
@@ -16,38 +16,45 @@
 
 from pithos.plugin import PithosPlugin
 from pithos.pithosconfig import get_data_file
-from mutagen.mp4 import MP4
 import os
 from shutil import copyfile as copy
-from glob import glob
+import urllib
+import eyed3
+import eyed3.id3
 
 class SavePlugin(PithosPlugin):
     preference = 'save'
-    _MUSIC_DIR = os.getenv("XDG_MUSIC_DIR")
+    _MUSIC_DIR = os.getenv("HOME") + '/Music/'
```

Comment:
What does the shell command `echo $XDG_MUSIC_DIR` return? I'm using `XDG_MUSIC_DIR` because not everyone puts their music in `~/Music`.